File: src/main/java/gestion_restaurant/entity/CommandeComplement.java
package gestion_restaurant.entity;

import lombok.*;
import java.math.BigDecimal;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@ToString
public class CommandeComplement {
    private Long id;
    private Commande commande;
    private Complement complement;
    private Integer quantite;
    private BigDecimal prixUnitaire;
}

File: Models/CommandeComplement.cs
namespace gestion_restaurant.Models
{
    public class CommandeComplement
    {
        public long Id { get; set; }

        public long? CommandeId { get; set; }
        public Commande Commande { get; set; }

        public long? ComplementId { get; set; }
        public Complement Complement { get; set; }

        public int Quantite { get; set; }
        public decimal? PrixUnitaire { get; set; }
    }
}
